woocommerce/wc-admin
# Packages: Version bumps & changelog maintenance

It looks like there's been some change to all the packages (whether package bumps or the textdomain change), so I'm going to bump all of them. This PR handles the versioning & changelog incrementing (some of which have been missed). Once merged, I'll run the publish command to push to npm.

**To test**

- Run `npm install` and build/start the project
- Check that there is no functionality change
- Check that the versions & changelogs look good
